bot/exts/moderation/infraction/infractions.py

```diff
@@ -402,10 +423,13 @@ async def action() -> None:
             # Skip members that left the server
             if not isinstance(user, Member):
                 return
+            duration_or_expiry = kwargs["duration_or_expiry"]
```

Comment:
Are you getting this from kwarg to avoid altering the signature ?

It's just that I think that if it's something that's cohesive with the command, it's worth changing the signature along with it

Reply:
I think the idea was to make the `apply_x` functions all have the same signature. It's not necessarily the best, but I'm not sure I want to start messing with it.

Reply:
Hummm, I guess we could leave it then, I don't think apply_timeout would be used somewhere else so that's probs fine